src/reporting/ReportingWindow.sol

```diff
@@ -289,4 +291,22 @@ contract ReportingWindow is DelegationTarget, Typed, Initializable {
         numberOfReportsByMarket[_market] += 1;
         return true;
     }
+
+    function getWinningPayoutDistributionHashFromFork(Market _market) constant returns (bytes32) {
```

Comment:
Yea, there were a few small functions toward the end of the reduction where I just moved them into another contract.